ansible/zuul.yaml

```diff
@@ -175,6 +175,8 @@
                 - pkg-map
                 - infra-package-needs
                 - zuul-worker
+                - dib-python
+                - ensure-venv
```

Comment:
This is already a dependency of infra-package-needs, so kind of redundant. Doesn't hurt to make it explicit, but I'd place it in front of that element, then.